SEC/Debt/AssetBackedSecurities/MortgageBackedSecurities.rdf

```diff
@@ -449,7 +437,7 @@ Consensus:Review.</skos:editorialNote>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-sec-dbt-ab-mbs;PassThroughMBSDeal">
```

Comment:
We need to revisit the whole term "deal" which I cannot see used outside FIBO. "Deal" seems to imply "contract" which si not at all the same as an offering which does not become a contract until the offer is taken/purchased

Reply:
@rivettp - I agree, but didn't want to make more changes via this issue than necessary.

Reply:
Agreed - it is a separate issue and I'm OK with deferring.